docs/guides/building-forms.md

```diff
@@ -237,9 +233,9 @@ If you submit this form, it will submit the data to the forms API endpoint `/api
 
 ![form-no-js](https://assets.vercel.com/image/upload/dpr_auto,q_auto,f_auto/nextjs/guides/building-forms/form-no-js.jpg)
 
-## Part 3: Forms in Next.js
+## Part 4: Configuring Forms in Next.js
 
-You have created a server with Vercel via Serverless Functions. Now it's time to configure the client (the form itself) inside Next.js using React. The first step will be extending your knowledge of HTML forms and converting it into React (using [JSX](https://reactjs.org/docs/introducing-jsx.html)).
+You have created a back-end server for form submission. Now it's time to configure the client (the form itself) inside Next.js using React. The first step will be extending your knowledge of HTML forms and converting it into React (using [JSX](https://reactjs.org/docs/introducing-jsx.html)).
```

Comment:
Im not sure this is correct. We've created an API Route (which run as a serverless function) but not a back-server

Reply:
Done.